scripts/CompareFitMinimizers/fitting_benchmarking.py

```diff
@@ -336,38 +346,36 @@ def run_fit(wks, prob, function, minimizer='Levenberg-Marquardt', cost_function=
     @returns the fitted parameter values and error estimates for these
     """
     status = None
-    chi2 = None
     param_tbl = None
-    fit_wks = None
     try:
         # When using 'Least squares' (weighted by errors), ignore nans and zero errors, but don't
         # ignore them when using 'Unweighted least squares' as that would ignore all values!
         ignore_invalid = cost_function == 'Least squares'
-
         # Note the ugly adhoc exception. We need to reconsider these WISH problems:
         if 'WISH17701' in prob.name:
             ignore_invalid = False
-        status, chi2, covar_tbl, param_tbl, fit_wks = msapi.Fit(function, wks, Output='ws_fitting_test',
-                                                                Minimizer=minimizer,
-                                                                CostFunction=cost_function,
-                                                                IgnoreInvalidData=ignore_invalid,
-                                                                StartX=prob.start_x, EndX=prob.end_x)
+
+        status = msapi.Fit(function, wks, Output='ws_fitting_test',
```

Comment:
I think I agree better with capture result from fit in one variable rather than individually. But if you do this do perhaps the return variable should be result or fit_result rather than status?

Reply:
yeah, I can change the variable name. I just picked status as it was the name I found in an example